### PR TITLE
Consider CRLF when splitting code lines

### PR DESCRIPTION
**Pull request by notZaki**
I noticed that code lines ending with `# hide` were not getting hidden in the output. 
The problem went away if I switched from CRLF to LF line endings.
This might be related to issue #613

I think the issue is caused by `split(code, '\n')` (line 680 in expanders.jl) which assumes LF line ending. 
For CRLF input, code containing `... # hide` gets split to `... # hide\r` and this causes the regex on line 681 to not match---this is because `$` does not match `\r` [[ref](https://discourse.julialang.org/t/regex-doesnt-match-end-of-line-with-both-carriage-return-and-line-feed/5795)]

This PR fixes that by splitting code with `r"\r?\n"`.

---

Minimum working example:

- make.jl
```
using Documenter
makedocs(sitename="Example")
```
- src/index.md (using CRLF)
```
# Example

```@example
import Random # hide
Random.seed!(1) # hide
A = rand(3, 3)
b = [1, 2, 3]
A \ b
Random.seed!(2); nothing # hide
```

- Ouput using master (note that last line still gets hidden correctly):
image

- Output using PR
image

- If LF is used, then both master and PR have the same output where code is hidden correctly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +55,8 @@
 
 * ![Bugfix][badge-bugfix] Tables now honor column alignment in the HTML output. If a column does not explicitly specify its alignment, the parser defaults to it being right-aligned, whereas previously all cells were left-aligned. ([#511][github-511], [#989][github-989])
 
+* ![Bugfix][badge-bugfix] Code lines ending with `# hide` are now properly hidden for CRLF inputs ([#991][github-991])
```
Comment (mortenpi) [inline CHANGELOG.md]:
```suggestion
* ![Bugfix][badge-bugfix] Code lines ending with `# hide` are now properly hidden for CRLF inputs. ([#991][github-991])
```

**Comment by mortenpi** [comment]
Some parallel processing here :laughing: Thanks! Will merge when tests go green.

**Comment by notZaki** [comment]
Update changelog.
Didn't know if the policy was to squash commits or not, so I forced push the changes.

**Comment by mortenpi** [comment]
I squash merge PRs usually, so no need to squash them by hand.

**Pull request merged**